src/BlazorWebView/src/Maui/Extensions/UriExtensions.cs

```diff
@@ -1,20 +1,30 @@
 ﻿using System;
 using System.IO;
 
-namespace Microsoft.AspNetCore.Components.WebView.Maui
+namespace Microsoft.AspNetCore.Components.WebView.Maui;
+
+internal static class UriExtensions
 {
-	internal static class UriExtensions
+	internal static bool IsBaseOfPage(this Uri baseUri, string? uriString)
 	{
-		internal static bool IsBaseOfPage(this Uri baseUri, string? uriString)
-		{
-			if (Path.HasExtension(uriString))
-			{
-				// If the path ends in a file extension, it's not referring to a page.
-				return false;
-			}
+		if (string.IsNullOrWhiteSpace(uriString))
+			return false;
```

Comment:
I think people here prefer curly braces. There are a few other places.
```suggestion
		if (string.IsNullOrWhiteSpace(uriString))
		{
			return false;
		}
```

Reply:
Thank you for the feedback.